.travis.yml

```diff
@@ -4,9 +4,11 @@ dist: xenial
   
 # Only clone the most recent commit.
 git:
-  depth: 1
+  depth: false
```

Comment:
Do we need this so it can analyse the code coverage and issues of everything in the PR, rather than just the last commit?

Reply:
Yep exactly